frontend/packages/ux-editor-v3/src/hooks/mutations/useAddLayoutMutation.ts

```diff
@@ -29,7 +30,9 @@ export const useAddLayoutMutation = (org: string, app: string, layoutSetName: st
 
   const save = async (updatedLayoutName: string, updatedLayout: IInternalLayout) => {
     const convertedLayout: ExternalFormLayoutV3 = internalLayoutToExternal(updatedLayout);
-    return await saveFormLayoutV3(org, app, updatedLayoutName, layoutSetName, convertedLayout);
+    return await saveFormLayoutV3(org, app, updatedLayoutName, layoutSetName, {
```

Comment:
We might consider just using the same function for v3 and v4 since they are now identical. But not critical by any means 😊

Reply:
Good point, but i was less sure if that statement will stay true when changing the backend next time?

Reply:
Then the (hopefully existing) playwright tests would fail 😇 
But I buy that point, and it's not a big issue. Let's leave it for now!